types/start-server-webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for start-server-webpack-plugin 2.2
+// Project: https://github.com/ericclemmons/start-server-webpack-plugin
+// Definitions by: AGenson <https://github.com/AGenson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Typescript Version: 3.8
+
+import { Plugin, Compiler } from 'webpack';
+
+export = StartServerWebpackPlugin;
+
+declare class StartServerWebpackPlugin extends Plugin {
+    /**
+     * @param name - Name of the server to start (built asset from webpack).
+     */
+    constructor(name: string);
```

Comment:
because of this:
https://github.com/ericclemmons/start-server-webpack-plugin/blob/master/src/StartServerPlugin.js#L5
this probably should read:
```suggestion
    constructor(name?: string);
```
so effectively creating parameterless constructor suppport.